sys/include/net/gnrc/ipv6.h

```diff
@@ -11,10 +11,63 @@
  * @ingroup     net_gnrc
  * @brief       GNRC's IPv6 implementation
  *
- * The IPv6 control thread understands messages of type
- *
- *  * @ref GNRC_NETAPI_MSG_TYPE_RCV, and
- *  * @ref GNRC_NETAPI_MSG_TYPE_SND,
+ * This module is for usage with the @ref net_gnrc_netapi
+ *
+ * # NETAPI command documentation
+ *
+ * ## `GNRC_NETAPI_MSG_TYPE_RCV`
+ *
+ * @ref GNRC_NETAPI_MSG_TYPE_RCV expects a @ref net_gnrc_pkt (referred to as
+ * "packet" in the following) in receive order (payload first, headers in
+ * decreasing order by layer). It should at least contain a link-layer payload.
```

Comment:
Does this mean: "The link-layer must contain a non-empty payload"?

Reply:
No. Will try to clarify.

Reply:
Then maybe put the stronger phrase: "The link-layer must contain at least a valid IPv6 header".

Reply:
Done